nukkit/src/main/java/net/craftingstore/nukkit/inventory/InventoryBuilder.java

```diff
@@ -0,0 +1,58 @@
+package net.craftingstore.nukkit.inventory;
+
+import cn.nukkit.inventory.Inventory;
+import cn.nukkit.item.Item;
+import cn.nukkit.utils.TextFormat;
+import com.nukkitx.fakeinventories.inventory.ChestFakeInventory;
+import net.craftingstore.core.models.api.inventory.CraftingStoreInventory;
+import net.craftingstore.core.models.api.inventory.InventoryItem;
+import net.craftingstore.nukkit.CraftingStoreNukkit;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class InventoryBuilder {
+    private CraftingStoreNukkit instance;
+
+    public InventoryBuilder(CraftingStoreNukkit instance) {
+        this.instance = instance;
+    }
+
+    public Inventory buildInventory(CraftingStoreInventory csInventory) {
+        return buildInventory(csInventory, null);
+    }
+
+    public Inventory buildInventory(CraftingStoreInventory csInventory, CraftingStoreInventoryHolder parent) {
+        String title = csInventory.getTitle();
```

Comment:
The title of the inventory is still Chest for me.

Reply:
Might be caused by the FakeInventories library not being able to register its event, might be fixed with latest commit

Reply:
Unless the name the CSInventory is getting is somehow null, it defaults to Chest